goth/runner/download/__init__.py

```diff
@@ -82,107 +83,35 @@ def _create_cache_dir(self, asset_id: str) -> Path:
         asset_path.mkdir(exist_ok=True, parents=True)
         return asset_path
 
-    def _parse_link_header(self, header_value: str) -> dict:
```

Comment:
replaced this with https://ghapi.fast.ai/page.html#paged, we can also speed up the process by using https://ghapi.fast.ai/page.html#pages (which create multi thread)